src/cluster.h

```diff
@@ -422,7 +422,7 @@ void slotToChannelAdd(sds channel);
 void slotToChannelDel(sds channel);
 void clusterUpdateMyselfHostname(void);
 void clusterUpdateMyselfAnnouncedPorts(void);
-sds clusterGenNodesDescription(int filter, int use_pport);
+sds clusterGenNodesDescription(int filter, int use_pport, int include_aux_fields);
```

Comment:
```suggestion
sds clusterGenNodesDescription(int filter, int use_pport, int verbose);
```
A thought to make the intention clearer. In the future it might just be more than the aux fields.